src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const status = person.sex === 'm'
```

Comment:
destructure your `person` object at the very start to avoid writing `person` everytime